datasource/get_watertemp.go

```diff
@@ -77,3 +77,39 @@ func WaterTemperatureSchemaToString(wts WaterTemperatureSchema) string {
 func hasLowerTemperature(location1 WaterTemperatureSchema, location2 WaterTemperatureSchema) bool {
 	return location1.Temp_water < location2.Temp_water
 }
+
+/* TO DO: Need to discuss this part of the code!
```

Comment:
Very good with a summary "comment" - if you feel we have discussed it enough, please remove it and other excessive comments.

Reply:
Done